src/AppStore.ts

```diff
@@ -23,7 +23,7 @@ export class AppStore {
                 sendSentryMessage('ERRORHANDLER:' + error);
             } catch (ex) {}
 
-            if (error.status && /400|500/.test(error.status)) {
+            if (error.status && /400|500|403/.test(error.status)) {
```

Comment:
really we should just handle everything that bubbles up here but i'm not quite ready to do so because there are some endpoints which say 404 to indicate no match (e.g quick search).  So, need to do some more testing